src/Program.cs

```diff
@@ -55,7 +54,7 @@ private static byte[] ComputeSignature(Archive archive)
                 reportedValue.Value.ToSignatureString(),
                 archive.Metadata.Created.ToSignatureString(),
                 archive.Metadata.TerminalIdentification,
-                operationName,
+                archive.Metadata.ArchiveType,
```

Comment:
Just realized this doesn't work for v1. It needs to be done in a backwards compatible way.